sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/StorageHelperTests.cs

```diff
@@ -11,17 +12,53 @@ namespace Azure.Monitor.OpenTelemetry.Exporter.Tests
 {
     public class StorageHelperTests
     {
-        // The directory separator is either '/' or '\' depending on the Platform, making unit testing tricky.
-        private static readonly char ds = Path.DirectorySeparatorChar;
+        private readonly string _testStorageDirectory;
+
+        public StorageHelperTests()
+        {
+            _testStorageDirectory = RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
+                ? "C:\\UnitTest"
+                : "/var/UnitTest";
```

Comment:
Who would be responsible for the clean up?

Reply:
No assumptions (that drive exists) and no cleanup.
In this test class, the platform has been mocked and fully testing the business logic.